gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -329,13 +329,9 @@ bool AREImporter::ChangeMap(Map *map, bool day_or_night)
 	// Small map for MapControl
 	ResourceHolder<ImageMgr> sm = GetResourceHolder<ImageMgr>(TmpResRef);
 
-	// night small map is *optional*!
-	if (!sm) {
-		//fall back to day minimap
-		sm = GetResourceHolder<ImageMgr>(map->WEDResRef);
-	}
-	
 	if (sm) {
+		// night small map is *optional*!
+		// keep the exising map if this one is null
```

Comment:
Doesn't this break if you start in an area at night?

Reply:
I dont think so. If you start at night then you should already have the night map (or normal map if there is no night). My understanding was there is _always_ a day map, just not always a night map.

Reply:
In the WED sense, yes, but it remains to be shown for the minimap. So it's easier to reinstate the fallback.

Reply:
I dont understand. if `sm` is found, great. if its not just keep the existing one. why is that broken? this is different code than loads the map initially.